zdns/main.go

```diff
@@ -59,6 +60,8 @@ func main() {
 	flags.BoolVar(&gc.TCPOnly, "tcp-only", false, "Only perform lookups over TCP")
 	flags.BoolVar(&gc.UDPOnly, "udp-only", false, "Only perform lookups over UDP")
 	servers_string := flags.String("name-servers", "", "comma-delimited list of DNS servers to use")
+	localaddr_string := flags.String("local-addr", "", "cp,,adeöo,oted öost pf local addresses to use")
```

Comment:
Something got garbled here. I'm not sure what this is supposed to say.

Reply:
Most likely one key right of what i intended to type... :see_no_evil: